packages/babel-plugin-transform-runtime/test/fixtures/runtime-corejs2/aliased-constructors/output.js

```diff
@@ -0,0 +1,14 @@
+var _Map = require("@babel/runtime-corejs2/core-js/map");
```

Comment:
This is just a nit and maybe not worth looking into, but do we want to just make the path `"@babel/runtime-corejs2/map"`?

Reply:
Hmm, I don't love having `core-js` repeated twice, but I do think having the core-js stuff grouped into a folder is nice. Do you just want to make it shorter, or is the repetition the part you want to avoid? Thoughts on other names?